### ISSUE TITLE
ZabbixAPI maintenance.delete causes error

### ISSUE THREAD

**Issue by timyllym** [grafana/grafana-zabbix#1178]
We have built our custom Grafana panels where we utilize Zabbix datasource heavily. We call ZabbixAPI directly through your datasource. 

Now after recent update when we call:
zabbix.zabbixAPI.request('maintenance.delete', [maintenanceID])

That causes error:
json: cannot unmarshal array into Go struct field ZabbixAPIResourceRequest.params of type map[string]interface {}

That's because maintenance.delete call takes only an array while most ZabbixAPI-calls take normal json object and in the models.go file the type of ZabbixAPIResourceRequest is defined to be map[string]interface{}

** Screenshot **
![Screenshot 2021-03-17 at 12 49 22](https://user-images.githubusercontent.com/13029109/111456572-d60f6080-871f-11eb-9559-f067200873a9.png)

**Software versions**

| Grafana | Zabbix | Grafana-Zabbix Plugin |
| ------- | ------ | --------------------- |
| 7.3.1     | 5.0.5   | 4.0.2                           |

**Comment by alexanderzobnin** [comment]
Yes, in that case `ZabbixAPIResourceRequest.Params` should be more general, like `interface{}`. I think I anyway need this in future, so I'll change this.
